Packs/SophosCentral/Integrations/SophosCentral/SophosCentral_test.py

```diff
@@ -0,0 +1,992 @@
+import json
+from datetime import datetime
+from pytest import raises
+from CommonServerPython import DemistoException
+
+BASE_URL = 'https://api-eu02.central.sophos.com'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
+
+def load_mock_response(file_name: str) -> dict:
+    """
+    Load one of the mock responses to be used for assertion.
+
+    Args:
+        file_name (str): Name of the mock response JSON file to return.
+    """
+    with open(f'test_data/{file_name}', mode='r', encoding='utf-8') as json_file:
+        return json.loads(json_file.read())
+
+
+def test_sophos_central_alert_list_command(requests_mock) -> None:
+    """
+    Scenario: List alerts
+    Given:
+     - User has provided valid credentials.
+     - Headers and JWT token have been set.
+    When:
+     - sophos_central_alert_list is called.
+    Then:
+     - Ensure number of items is correct.
+     - Ensure the number of fields in each output is correct.
+     - Ensure outputs prefix is correct.
+     - Ensure the raw response matches the mocked API response.
+    """
+    from SophosCentral import Client, sophos_central_alert_list_command
+    mock_response = load_mock_response('alert_list.json')
+    requests_mock.get(f'{BASE_URL}/common/v1/alerts', json=mock_response)
+    mock_client_data = load_mock_response('client_data.json')
+    requests_mock.get('https://api.central.sophos.com/whoami/v1', json=mock_client_data)
+    client = Client(bearer_token='a', verify=False, client_id='a',
+                    client_secret='b', proxy=False)
+
+    result = sophos_central_alert_list_command(client, {'limit': '50'})
+    assert len(result.outputs) == 3
+    assert len(result.outputs[0]) == 14
+    assert result.outputs_prefix == 'SophosCentral.Alert'
+    assert result.raw_response == mock_response
+
+
+def test_sophos_central_alert_get_command(requests_mock) -> None:
+    """
+    Scenario: Get a single alert.
+    Given:
+     - User has provided valid credentials.
+     - Headers and JWT token have been set.
+    When:
+     - sophos_central_alert_get is called.
+    Then:
+     - Ensure the number of fields in each output is correct.
+     - Ensure outputs prefix is correct.
+     - Ensure the raw response matches the mocked API response.
+    """
+    from SophosCentral import Client, sophos_central_alert_get_command
+    mock_response = load_mock_response('alert_single.json')
+    alert_id = '56931431-9faf-480c-ba1d-8d7541eae259'
+    requests_mock.get(f'{BASE_URL}/common/v1/alerts/{alert_id}', json=mock_response)
+    mock_client_data = load_mock_response('client_data.json')
+    requests_mock.get('https://api.central.sophos.com/whoami/v1', json=mock_client_data)
+    client = Client(bearer_token='a', verify=False, client_id='a',
+                    client_secret='b', proxy=False)
+
+    result = sophos_central_alert_get_command(client, {'alert_id': alert_id})
+    assert len(result.outputs) == 14

```

Comment:
what does that mean? the number of fields?
not a must check in general.

Reply:
Removed from all tests